source/client/HandView.py

```diff
@@ -16,30 +16,25 @@ class HandView:
     def __init__(self, controller, display):
         self.controller = controller
         self.display = display
-        '''
-        # initialize pygame modules
-        pygame.init()
-        # initialize variables
-        self.Notification = "It is someone's turn."
-        '''
         self.current_hand = []
         self.last_hand = []
         self.hand_info = []          # will contain UICardWrapped elements of current_hand
+        self.prepared_cards = []     # will contain list of prepared cards from controller
```

Comment:
You shouldn't need to save the prepared_cards, the controller does, and when they are played the hand will be adjusted appropriately as for discard and draw. Since its not used anywhere I can remove it when I do my TODOs and wire up buttons